internal/driver/driver_test.go

```diff
@@ -125,6 +125,17 @@ func TestParse(t *testing.T) {
 				f.bools["topproto"] = true
 			}
 
+			// Filters should not be applied on first pprof invocation.
```

Comment:
Why? Is this a hack to make the old tests work or is it truly to right thing to do?

Reply:
I feel like the right thing may be to re-write this test so that we can specify flags for the first invocation of pprof (which would have to have -proto or -topproto  as format so a protobuf is written out), and then separate flags to 

This would make the test work for testing proto and topproto, without changing the test too much or duplicating a lot of this code. 

So, most current test cases would look like: 
```
{
  firstInvkFlags: "proto", 
  secondInvkFlags: "text,functions,flat", 
  profile: "cpu"
},
```

Or, to simplify writing test cases, we can have firstInvkFlags be set to "proto" or "proto,lines" when that value is an empty string.

What do you think of this?

Reply:
That seems overkill to me. I feel the first invocation serves only one purpose: check that the proto generation works as expected under different circumstances. Could we just run the first invocation without any flags?

Alternatively, it might be even cleaner if we'd have just one invocation, not two, and a separate test case for "proto" format, with whatever flags we want to test. The only wrinkle there is how to compare the output since proto is a binary file.